ptypes/regen.sh

```diff
@@ -54,18 +76,23 @@ done
 # Pass 2: copy files, making necessary adjustments.
 for up in "${!filename_map[@]}"; do
   f=${filename_map[$up]}
-  shortname=$(basename $f | sed 's,\.proto$,,')
+  shortname=$(basename $f | $(which_sed) 's,\.proto$,,')
   cat $tmpdir/$UPSTREAM_SUBDIR/$up |
     # Adjust proto package.
     # TODO(dsymonds): Upstream the go_package option instead.
```

Comment:
This TODO is in progress; I'd prefer to just wait for it to land, and then rip this out entirely.